helm-values.tf

```diff
@@ -1,12 +1,16 @@
 locals {
   elasticsearch_endpoint = "https://${element(concat(aws_elasticsearch_domain.es.*.endpoint, aws_elasticsearch_domain.public_es.*.endpoint), 0)}"
+
+  template_filename = "${substr("${path.module}/templates/helm-values.tpl.yaml", length(path.cwd) + 1, -1)}"
+  // +1 for removing the "/"
+  helm_values_filename = "${substr("${path.module}/helm-values.yaml", length(path.cwd) + 1, -1)}"
 }
 
 data "aws_region" "current" {}
 
 data "template_file" "helm_values" {
   count    = "${length(var.prometheus_labels) != 0 ? 1 : 0}"
-  template = "${file("${path.module}/templates/helm-values.tpl.yaml")}"
+  template = "${file("${local.template_filename}")}"
```

Comment:
why is this change needed? as far as I know template location doesn't result in a change of the plan.